debian/README.Debian

```diff
@@ -0,0 +1,6 @@
+distribution-gpg-keys for Debian
+----------------------------------------------------------
+
+<possible notes regarding this package - if none, delete this file>
+
+ -- Miroslav Suchý <msuchy@redhat.com>  Mon, 27 Jun 2022 17:47:42 +0000
```

Comment:
It would be great if the contact here would be someone who is responsible for the Debian packaging. You?